src/Client/Web/Bit.Client.Web.BlazorUI/Extensions/ObjectExtensions.cs

```diff
@@ -5,9 +5,14 @@ namespace Bit.Client.Web.BlazorUI
 {
     public static class ObjectExtensions
     {
-        public static string? GetValueFromProperty(this object? obj, string propertyName)
+        public static string? GetValueAsStringFromProperty(this object? obj, string propertyName)
         {
-            return obj?.GetType().GetProperty(propertyName)?.GetValue(obj)?.ToString();
+            return obj?.GetValueAsObjectFromProperty(propertyName)?.ToString();
+        }
```

Comment:
remove this redundant method and instead use the ToString method at the caller site directly.